Model/Page.php

```diff
@@ -174,6 +174,11 @@
     protected static $slugifyMethod;
 
     /**
+     * @var int
+     */
+    private $targetRedirectType = 302;
```

Comment:
@greg0ire i just installed this in another project but this `private` makes some problems which i don't really understand. Exception: `Property Sonate\PageBundle\Entity\BasePage::$targetRedirectType does not exist.`

As soon i change it back to protected everything works just fine. Maybe because the package is extended  (just a guess)? Unfortunately i don't have much time to look into that this week.

<img width="1060" alt="screen shot 2017-06-20 at 16 58 44" src="https://user-images.githubusercontent.com/321891/27340057-c5038a6c-55d9-11e7-9116-02a5961a7392.png">